clients/google-js/README.md

```diff
@@ -1,4 +1,4 @@
-# Google Cloudendpoints ID Token Javascript Client
+# CloudEndpoints JavaScript client using Google ID Token
```

Comment:
nit... Please add a space between Cloud Endpoints. The rest of it looks good to me.

Reply:
Done